packages/cli/src/commands/link.ts

```diff
@@ -1,21 +1,41 @@
 import push from './push';
 import link from '../scripts/link';
+import { promptIfNeeded } from '../utils/prompt';
 
 const name: string = 'link';
 const signature: string = `${name} [dependencies...]`;
 const description: string = 'links project with a list of dependencies each located in its npm package';
 
+const props = {
+  dependencies: {
+    type: 'input',
+    message: 'Provide an EVM-package name and version',
+    default: ['openzeppelin-eth']
```

Comment:
Let's somehow explain that the full format should be `name@version`

Reply:
Have a crazy idea :nerd_face:: we can run a `npm view openzeppelin-eth` to get the current package version and show `openzeppelin-eth@2.1.3` (or whichever the latest version is) instead of `openzeppelin-eth`. This way, we don't have to update the "example" version each time the openzeppelin team releases a new version.

Reply:
Isi pisi:
```ts
  public static async fetchVersionFromNpm(name: string): Promise<string | null> {
    const execAsync = promisify(exec);
    try {
      const { stdout }= await execAsync(`npm view ${name} | grep latest`);
      const versionMatch = stdout.match(/([0-9]*\.){2}[0-9]*/)
      if (versionMatch.length) return versionMatch[0];
    } catch(error) {
      return null;
    }
  }

```